contracts/src/lib/SupplySourceHelper.sol

```diff
@@ -46,12 +47,19 @@ library SupplySourceHelper {
     }
 
     /// @notice Locks the specified amount sent by the msg.sender into custody.
```

Comment:
Let's document the conditions under which this can revert (`NoBalanceIncrease`).

Reply:
Also mention that this can return a higher amount that requested if the token is inflationary.

Reply:
8f397bcd